definitions/v3/xthor-api.yml

```diff
@@ -0,0 +1,179 @@
+    - { id: 117, cat: PC, desc: "Logiciels/ROM" }
+    - { id: 21, cat: PC, desc: "Logiciels/Applis PC" }
+    - { id: 22, cat: PC/Mac, desc: "Logiciels/Applis Mac" }
+    - { id: 23, cat: PC/Mobile-Android, desc: "Logiciels/Smartphone" }
+
+  modes:
+    search: [ q ]
+    movie-search: [ q, tmdbid ]
+    tv-search: [ q, season, ep ]
+    book-search: [ q ]
+
+settings:
+  - name: passkey
+    type: password
+    label: Passkey
+  - name: freeleech
+    type: checkbox
+    label: Search freeleech only
+    default: false
+  - name: multilanguage
+    type: text
+    label: Replace MULTI by
+    default: MULTI
+  - name: subfrench
+    type: text
+    label: Replace "VOSTFR/SUBFRENCH" by
+    default:
+
+search:
+  paths:
+    # https://api.xthor.tk/?passkey=<redacted>&search=&category=&freeleech=&tmdbid=&size=&accent=&price=&page=
+    - path: "https://api.xthor.tk/"
+      inputs:
+        passkey: "{{ .Config.passkey }}"
+        category: "{{ if .Categories }}{{ range .Categories }}{{.}}+{{end}}{{ else }}{{ end }}"
+        search: "{{ if .Query.TMDBID }}{{ else }}{{ .Keywords }}{{ end }}"
+        tmdbid: "{{ .Query.TMDBID }}"
+        freeleech: "{{ if .Config.freeleech }}1{{ else }}{{ end }}"
+      response:
+        type: json
+
+  rows:
+    selector: torrents
+
+  fields:
+    id:
+      selector: id
+    category:
+      selector: category
+    title_original:
+      selector: name
+    title_multi:
+      text: "{{ .Result.title_original }}"
+      filters:
+        - name: re_replace
+          args: [ "(?i)[\\.\\s\\[\\-]multi[\\.\\s\\]\\-]", ".{{ .Config.multilanguage }}." ]
+    title_multi_out:
+      text: "{{ if .Config.multilanguage }}{{ .Result.title_multi }}{{ else }}{{ .Result.title_original }}{{ end }}"
+    title_vostfr:
+      text: "{{ .Result.title_multi_out }}"
+      filters:
+        - name: re_replace
+          args: [ "(?i)[\\.\\s\\[\\-]vostfr[\\.\\s\\]\\-]", ".{{ .Config.subfrench }}." ]
+        - name: re_replace
+          args: [ "(?i)[\\.\\s\\[\\-]subfrench[\\.\\s\\]\\-]", ".{{ .Config.subfrench }}." ]
+    title:
+      text: "{{ if .Config.subfrench }}{{ .Result.title_vostfr }}{{ else }}{{ .Result.title_multi_out }}{{ end }}"
+    details:
+      text: "https://xthor.tk/details.php?id={{ .Result.id }}"
+    download:
+      selector: download_link
+    tmdbid:
+      selector: tmdb_id
+    files:
+      selector: numfiles
+    date:
+      selector: added
+    size:
+      selector: size
+    seeders:
+      selector: seeders
+    leechers:
+      selector: leechers
+    grabs:
+      selector: times_completed
+    downloadvolumefactor:
+      # api returns 0=false, 1=true
+      selector: freeleech
+      case:
+        0: 1 # not free
+        1: 0 # freeleech
+      text: 0
+    uploadvolumefactor:
+      text: 1
+    minimumseedtime:
+      # 24 hours (as seconds = 1 x 24 x 60 x 60)
+      text: 86400
+    minimumratio:
+    # minimum ratio per torrent is 0.8
+      text: 0.8
```

Comment:
Nit - new line end of file
Nit - add the engine for the indexer

Reply:
By engine you mean which script the indexer use to run ? In this case it's a custom script made by them

Reply:
yup, look at the other ymls, there are a few with custom engines, i'd just follow a similar format to those